tests/test_api.py

```diff
@@ -1925,7 +1925,7 @@ def test_config_map(self, osbs, name, data):
             'apiVersion': 'v1',
             'kind': 'ConfigMap',
             'metadata': {'name': name},
-            'data': data,
+            'data': {key: json.dumps(value) for key, value in data.items()},
```

Comment:
Improved part of the code, which uses actual content of the of :
```diff
-        obj = {
-            'apiVersion': 'v1',
-            'kind': 'ConfigMap',
-            'metadata': {}
-        }
+        with open(os.path.join(INPUTS_PATH, "config_map.json")) as fp:
+            obj = json.load(fp)
```

Also we probably should be storing filename in a variable in `tests/constants.py` and use it in this test